### PR TITLE
Accept cookies functionality

### LINKED ISSUE #6: Accept Cookies functionality

### PR DESCRIPTION
**Pull request by martinbavio**
This PR adds simple cookie support to make the accept cookies functionality to work. 

Fixes https://github.com/vercel/commerce/issues/6.

### CODE REVIEW

**Comment by marbiano** [comment]
Added a commit to fix [this issue](https://github.com/vercel/commerce/issues/53)

**Comment by okbel** [comment]
@marbiano Could you check why the build is failling? Thank you!!

File: components/core/Featurebar/Featurebar.tsx [hunk 1]
```diff
@@ -1,27 +1,27 @@
 import cn from 'classnames'
-import { FC } from 'react'
+import styles from './FeatureBar.module.css'
```
Comment (okbel) [inline components/core/Featurebar/Featurebar.tsx:2]: Minor but for consistency we're using `import s from '..'`

**Comment by marbiano** [comment]
@okbel build fixed, sorry for the delay!

**Comment by marbiano** [comment]
@okbel just merged all the updates from master and resolved quite a few conflicts, do you mind to merge this one if it looks good to you?

File: package.json [hunk 2]
```diff
@@ -19,6 +19,8 @@
     "bowser": "^2.11.0",
     "classnames": "^2.2.6",
     "email-validator": "^2.0.4",
+    "intersection-observer": "^0.11.0",
```
Comment (okbel) [inline package.json:22]: y este?

Comment (okbel) [reply package.json:22]: who this

Comment (marbiano) [reply package.json]: Good catch, just removed it.

**Review by okbel — APPROVED** [review]
Amazing <3

**Comment by momoiicom** [comment]
Completed by Sandcastle

**Pull request merged**